package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@hilla/react-components",
-  "version": "2.4.0-alpha2",
+  "name": "@vaadin/react-components",
+  "version": "24.4.0-alpha2",
```

Comment:
The web component version computed in `package-json-version.ts` should probably no longer have the `22 +` [here](https://github.com/vaadin/react-components/blob/46925fb2714a672c5c292cad1fae84e18c44e66e/scripts/package-json-version.ts#L12).